src/Tribe/Views/V2/Hooks.php

```diff
@@ -179,4 +180,14 @@ public function filter_query_vars( array $query_vars = [] ) {
 
 		return $query_vars;
 	}
+
+	/**
+	 * Filters the body classes.
+	 * Add theme compatibility classes.
```

Comment:
Add a blank line here or, better, join to read "Filters the body classes to add..."